docs/index.rst

```diff
@@ -26,6 +27,13 @@ Carsus is a package to manage atomic datasets. It can read data from a variety o
     io/vald
     io/barklem2016
 
+.. toctree::
+    :maxdepth: 2
+    :caption: Reference
+    :hidden:
+    reference/notation.rst
+    reference/level_index_id_number.rst
```

Comment:
Check why this doesn't show up

Reply:
Unsure if this is the answer, but perhaps check if you need a linebreak before you specify the files.